components/TrackingOrderPage/TrackingOrderPage.tsx

```diff
@@ -50,7 +48,7 @@ const TrackingOrderPage = () => {
       const date = await getServerDate()
       let ordersClone = clone(orders)
       if (ordersClone) {
-         ordersClone.forEach(order => {
+         ordersClone.forEach((order) => {
```

Comment:
Avoid making unnecessary changes. Changes should have reasons to be made, that's why git keeps the history of everything, so they can be referred to later on if needed.